v19.1/gssapi_authentication.md

```diff
@@ -0,0 +1,106 @@
+---
+title: GSSAPI Authentication (Enterprise)
+summary: Learn about the GSSAPI authentication features for secure CockroachDB clusters.
+toc: true
+---
+
+CockroachDB supports the Generic Security Services API (GSSAPI) with Kerberos authentication as an [Enterprise feature](enterprise-licensing.html).
+
+## Configuring KDC
+
+To use Kerberos authentication with CockroachDB, configure a Kerberos service principal name (SPN) for CockroachDB and generate a valid keytab file with the following specifications:
+
+- Set the SPN to `postgres`.
+- Create SPNs for all DNS addresses that a user would use to connect to your CockroachDB cluster (including any TCP load balancers between the user and the CockroachDB node) and ensure that the keytab contains the keys for every SPN you create.
+
+## Configuring the CockroachDB node
+1. Copy the keytab file at a location accessible by the cockroach binary.
+
+2. [Create certificates](create-security-certificates.html) for internode and root user authentication:
```

Comment:
Does it make more sense to link to another part of the docs here, instead of duplicate how to create certs? I kind of suspect most people at this step already have their certs. Just a question, since you may already have a policy/decision about this.